addons/stock/static/src/views/list/inventory_report_list_controller.js

```diff
@@ -19,11 +19,12 @@ export class InventoryReportListController extends ListController {
             (!this.props.context.inventory_mode || this.props.context.inventory_report_mode)
         ) {
             // hack so we don't show some of the default actions when it's inappropriate to
-            const { print, action } = actionMenus;
-            return {
-                action: action.filter((a) => a.name !== "Set"),
-                print: print.filter((a) => a.name !== "Count Sheet"),
-            };
+            const {print, action} = actionMenus;
+            return Object.assign(
+                {},
+                print.filter(a => a.name !== this.env._t('Count Sheet')),
+                action.filter(a => a.name !== this.env._t('Set')),
+            );
```

Comment:
:crying_cat_face: You need to keep the changes from https://github.com/odoo/odoo/commit/96e158a8b8bf3d9adf8bc634cf690c7f3a0cf829#diff-62dee5fef0aafc92caf8332c8352b4eef9873c6dec12dbf2f1970d3a2395f338 in your commit, i.e. don't re-add in the stuff it deleted (i.e. `Object.assign()`) + don't lose the stuff it added (i.e. `action: ` and `print: `)

Also ideally don't add in unnecessary formatting changes if they don't add anything (i.e. removing the spacing before `}` and after `{` in `const { print, action } = actionMenus;`)

```suggestion
                action: action.filter(a => a.name !== this.env._t('Set')),
                print: print.filter(a => a.name !== this.env._t('Count Sheet')),
```

Reply:
@ticodoo is it some rule not to make such changes after a REF is done? even for formatting?

Reply:
There's no hard rule, but in this case you were re-adding in old code which was possibly either:

- not compatible with the refactoring changes (I'm no JS expert, but they probably made the change in the refactoring either to make sure it's compatible with their other changes )
- OR  purposely changed to make the code cleaner according to the JS team's formatting standard

Either way, since the refactoring was done by the JS team, you can assume it matches their expected formatting styling and we shouldn't undo it so we can stay as consistent with their style as possible. I'm not sure if you have it set up, but there is also a script that should auto-enforce their styling expectations in master branches: https://github.com/odoo/odoo/wiki/Javascript-coding-guidelines#tooling

Reply:
Note that you might have to install npm for their formatting tooling to work if you don't already have it installed.